.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report an issue with the .NET Project System for Visual Studio
+title: 
+labels: 
```

Comment:
Please add `Bug` as an automatic label.

Reply:
I considered that but decided against it as I suspect we'll see a lot of stuff filed that isn't actually a bug. Let's do this in triage for things we actually believe are real bugs.

If that approach creates undue work I'm fine with adding the automatic label then.